Index: src/App.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React, { Component } from 'react';\r\nimport logo from './logo.svg';\r\nimport './App.css';\r\n\r\nclass App extends Component {\r\n  render() {\r\n    return (\r\n      <div className=\"App\">\r\n        <header className=\"App-header\">\r\n          <img src={logo} className=\"App-logo\" alt=\"logo\" />\r\n          <p>\r\n            Edit <code>src/App.js</code> and save to reload.\r\n          </p>\r\n          <a\r\n            className=\"App-link\"\r\n            href=\"https://reactjs.org\"\r\n            target=\"_blank\"\r\n            rel=\"noopener noreferrer\"\r\n          >\r\n            Learn React\r\n          </a>\r\n        </header>\r\n      </div>\r\n    );\r\n  }\r\n}\r\n\r\nexport default App;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/App.js	(revision dda664c06006554db46f6ad2375cfbb5a98c7976)
+++ src/App.js	(date 1547057933228)
@@ -1,25 +1,29 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import css from "./styles/app.module.css";
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
+      <div className={css.layout}>
+        <header className={css.header}>
+          <b className={css.title}>IntelliJ IDEA 2017.3! Help</b>
         </header>
+        <div className={css.app}>
+          <div className={css.nav}>nav</div>
+          <div className={css.content}>
+            Morbi malesuada dui dui, ut lacinia elit malesuada id. Quisque eu
+            leo faucibus, posuere dui sit amet, aliquet tortor. Suspendisse
+            mauris lacus, rhoncus quis sollicitudin in, bibendum a quam. Donec
+            cursus urna risus, sit amet faucibus lacus facilisis at. Phasellus
+            mollis tempor velit ac finibus. Morbi volutpat porttitor dolor vitae
+            finibus. Proin pellentesque dignissim metus, non rutrum arcu blandit
+            vitae. Sed commodo mauris et dignissim euismod. In pulvinar sodales
+            purus. Nulla mollis elit urna, lacinia tincidunt mi aliquam id. Sed
+            vel ex facilisis, placerat orci id, pulvinar odio. Maecenas
+            fermentum massa quis libero hendrerit commodo. Proin nisl sapien,
+            aliquam at leo id, vulputate varius lectus. Vivamus at egestas urna.
+          </div>
+        </div>
       </div>
     );
   }
